### PR TITLE
Refactor Displayer

### PR DESCRIPTION
**Pull request by onmyway133**
- This make Imaginary work on any View. The difference is the displayer. User can customise it to  be on any views or custom view they want
- We provide 3 default displayers, for imageview, for button image, for button background image.

### CODE REVIEW

File: ImaginaryTests-iOS/Button+ImaginaryTests.swift [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        }
+    })
+
+    // Set to placeholder initially
+    XCTAssertEqual(button.imageView?.image?.size, CGSize(width: 50, height: 50))
+
+    wait(for: [expectation], timeout: 1)
+  }
+}
+
```
Comment (vadymmarkov) [inline ImaginaryTests-iOS/Button+ImaginaryTests.swift]: Too many lines 😄

**Review by vadymmarkov — APPROVED** [review]
👍

File: ImaginaryTests-iOS/Button+ImaginaryTests.swift (hunk 1 above)
Comment (onmyway133) [reply ImaginaryTests-iOS/Button+ImaginaryTests.swift]: 😬

**Comment by onmyway133** [comment]
@vadymmarkov ❤️

**Pull request merged**